### ISSUE TITLE
UX - changing the language in the author area

### ISSUE THREAD

**Issue by mkduew** [trendschau/typemill#446]
After changing the language in the author area, a message appears "Settings have been saved".

If the language has changed, there should be also a notice, that the user has to reload the page to use the new language.

<img width="900" alt="screenshot" src="https://github.com/typemill/typemill/assets/17476152/d4ce56af-f463-4773-ab99-da8bc59dc9a6">

Typemill version 2.2.1

**Comment by trendschau** [comment]
I agree but this is technically hard to implement, I have to listen to this specific change and trigger a page refresh. Possible, but with quite some extracode that makes the codebase even more complex, I want to avoid this and favour clean and easy code over user experience in this case.
